src/frontend/screens/Observation/index.tsx

```diff
@@ -61,7 +66,13 @@ export const ObservationScreen: NativeNavigationComponent<'Observation'> = ({
   const {lat, lon, createdBy} = observation;
   const isMine = deviceId === createdBy;
   // Currently only show photo attachments
```

Comment:
This comment isn't respected. Can we make sure to only show photo attachments?

Reply:
Good point, that's sorted now 👍